test/run

```diff
@@ -12,6 +12,11 @@ if [ "$TEST_SCENARIO" == "devel" ]; then
     export TEST_COVERAGE=yes
 fi
 
+# Workaround broken podman pasta on Fedora-CoreOS https://bugzilla.redhat.com/show_bug.cgi?id=2277954
+if [ "$TEST_OS" == "fedora-coreos" ]; then
+    export RUN_TESTS_OPTIONS+=" --enable-network"
```

Comment:
That "+" syntax doesn't work here. Please do the `="$RUN_TESTS_OPTIONS --enable-network"` shell approach.

Reply:
Curiously it does? 

```
TEST_AUDIT_NO_SELINUX=1 test/common/run-tests --track-naughties --enable-network
```